tests/client.rs

```diff
@@ -0,0 +1,25 @@
+use http::Request;
+use lightarti_rest::Client;
+
+mod utils;
+
+#[tokio::test]
+pub async fn test_get() {
```

Comment:
What does this test need with regard to setup? I suppose it needs the directory-information somewhere - is it documented how to run the test? Is it somewhere in the Makefile?